sdk/rms_sdk/GoogleTest/SConscript

```diff
@@ -0,0 +1,12 @@
+from build_config import *
+
+Import('rms_env')
+
+rms_env.Append(CPPDEFINES = ['QTFRAMEWORK'])
+
```

Comment:
you don't need this file.
it is replaced by #googletest\SConscript

Reply:
Removed sdk/rms_sdk/GoogleTest